healthtools_ec/initiates.py

```diff
@@ -25,41 +32,37 @@ def initiates_home():
             db.session.commit()
             response = email_initiates(initiate)
             print(response)
-            if session['lang']:
-                return render_template('initiates/initiateredirect_xh.html')
+            if session["lang"]:
+                return render_template("initiates/initiateredirect_xh.html")
             else:
-                return render_template('initiates/initiateredirect.html')
+                return render_template("initiates/initiateredirect.html")
         else:
-            if request.is_xhr:
-                status = 412
+            if session["lang"]:
+                flash("Please correct the problems below and try again.", "warning")
             else:
-                if session['lang']:
-                    flash('Please correct the problems below and try again.', 'warning')
-                else:
-                    flash('Please correct the problems below and try again.', 'warning')
-
-    if not request.is_xhr:
-        if session['lang']:
-            resp = make_response(render_template('initiates/initiates_xh.html', form=form))
-        else:
-            resp = make_response(render_template('initiates/initiates.html', form=form))
-
+                flash("Please correct the problems below and try again.", "warning")
```

Comment:
Aren't we doing the same think on `if` and `else`

Reply:
We are using different `template` depending on the `lang` but I'm updating this to what @kilemensi  suggested [here](https://github.com/CodeForAfrica/HealthTools.ZA-Ezolwaluko/pull/2/files/fb9a92d0db03f5f166430bc7ceb617782e41db5d#r843694206)

Reply:
@esirK I speaking about line `40, 41, 42 and 43`. Else of form.validate()

it reads from 
```
if form.validate():
   ...
   ...
else:
   if session["lang"]:
      flash("Please correct the problems below and try again.", "warning")
   else:
      flash("Please correct the problems below and try again.", "warning")
```

Reply:
Oooh, I see. I think the intent is to have an `XHOSA` version of the response. I'll look for a translation although I think adding a localization might be better.